app/testOnly/views/LoginPage.scala.html

```diff
@@ -19,34 +19,52 @@
 @import views.html.layouts.unifiedLayout
 
 @this(
-    layout: unifiedLayout,
-    govukButton : GovukButton,
-    govukSelect : GovukSelect,
-    govukCheckboxes : GovukCheckboxes,
-    form: FormWithCSRF
+        layout: unifiedLayout,
+        govukButton: GovukButton,
+        govukSelect: GovukSelect,
+        govukCheckboxes: GovukCheckboxes,
+        form: FormWithCSRF
 )
 
-@(postAction: Call, users: Seq[UserRecord])(implicit request : Request[_], messages: Messages)
+@(postAction: Call, users: Seq[UserRecord])(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = "Login - Manage your Income Tax updates - GOV.UK") {
-<link href='/report-quarterly/income-and-expenses/view/assets/stylesheets/application.css' media="screen" rel="stylesheet" type="text/css" />
+    <link href='/report-quarterly/income-and-expenses/view/assets/stylesheets/application.css' media="screen" rel="stylesheet" type="text/css" />
+
+    <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
```

Comment:
The script with
function enableOptOutDropdowns in it? How can I do this?

Reply:
in the worst case you can add copy of this script to the project in javascript folder, but we need to make sure this would not be exposed to outside of testOnly / QA / staging env